dirsrvtests/tests/suites/export/export_test.py

```diff
@@ -106,23 +101,20 @@ def test_db2ldif_cli_with_non_accessible_ldif_file_path(topo):
 
     log.info("Performing an offline export to a non accessible ldif file path - should fail properly")
     try:
-        subprocess.check_call([db2ldif_cmd, topo.standalone.serverid, 'db2ldif', 'userroot', export_ldif])
+        # subprocess.check_call([db2ldif_cmd, topo.standalone.serverid, 'db2ldif', 'userroot', export_ldif])
```

Comment:
I think this comment line can be safely removed as it mostly repeats the following.

Reply:
Fixed